README.md~

```diff
@@ -0,0 +1,39 @@
+# TheReadingMachine
+A Mean, Lean, Reading Machine
```

Comment:
Remove the emacs spurious files.
Maybe add the following line to .gitignore
*~